core/src/main/java/io/lionweb/language/Concept.java

```diff
@@ -92,16 +92,26 @@ public boolean isAbstract() {
     return this.getPropertyValue("abstract", Boolean.class, false);
   }
 
-  public void setAbstract(boolean value) {
+  public @Nonnull Concept setAbstract(boolean value) {
     this.setPropertyValue("abstract", value);
+    return this;
+  }
+
+  public @Nonnull Concept makeAbstract() {
```

Comment:
how about making this a parameterless overload of `setAbstract()`, effectively having a default parameter value of `true`?

Reply:
Makes sense

Reply:
Renaming also `makePartition` to `setPartition`